### DISCUSSION TITLE
Do you mess around with posts for mastercam?

### DISCUSSION THREAD

**Discussion by HateWhatOwnsYou** [MarkRoldan88/Mastercam-Level-Exporter#60] [General]
Title has everything in it that I’m wondering lol

**Comment by MarkRoldan88** [comment]
Hey @HateWhatOwnsYou , 

I've worked with them in the _past_ (it's been a minute), but I currently don't work on any kind of posts or post modifications in Mastercam. Plus I **really** don't enjoy the Mastercam post language.

Thanks
